.github/workflows/.github/workflows/dependency-audit.yml

```
@@ -0,0 +1,132 @@
+            let summary = `🔎 Dependency audit artifacts available:\\n\\n`;
+            const files = fs.readdirSync(path);
+            files.forEach(f => summary += `- ${f}\\n`);
+            github.rest.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number || github.context.payload.pull_request.number,
+              body: summary
+            });
+
+      - name: Finish
+        run: echo "Artifacts uploaded: $(ls -la artifacts || true)"
```

actionlint (1.7.11)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)